tox.ini

```diff
@@ -73,12 +73,10 @@ setenv =
   {[testenv]setenv}
```

Comment:
This had merge conflicts. I messed it up in the first commit, but I think it's now right.